git-d8ci

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Do you use `sh` at all? If not, symlink `/bin/sh` to `/bin/bash` and you don't need to change userspace scripts.